test/file.js

```diff
@@ -96,6 +96,14 @@ describe('file', function () {
 				done();
 			});
 		});
+
+		it('should error if folder is relative', function (done) {
+			file.saveFileToLocal(filename, '../../text', tempPath + '000000000', function (err) {
+				assert(err);
+				assert.strictEqual(err.message, '[[error:invalid-path]]');
+				done();
+			});
+		});
```

Comment:
Advise adding a test to ensure that `saveFileToLocal` also fails if absolute path does not start with `upload_path`